src/guides/v2.3/config-guide/multi-site/ms_nginx.md

```diff
@@ -1,6 +1,6 @@
 ---
 group: configuration-guide
-title: Tutorial&mdash;Set up multiple websites or stores with nginx
+title: Tutorial—Set up multiple websites or stores with nginx
```

Comment:
Revert this. Long term, it would be better if the title were changed so that that there's no em dash.

Reply:
**done**